sensor/kubernetes/eventpipeline/output/output.go

```diff
@@ -14,6 +16,8 @@ func New(detector detector.Detector, queueSize int) component.OutputQueue {
 		detector:     detector,
 		innerQueue:   ch,
 		forwardQueue: forwardQueue,
+		stopped:      &atomic.Bool{},
 	}
+	outputQueue.stopped.Store(false)
```

Comment:
Isn't the default false? If yes then I reckon you don't need to do this

Reply:
Good call. I've checked and false is the default so no need to initialize.